manifests/vhost.pp

```diff
@@ -2832,10 +2832,10 @@
     $security_params = {
       'modsec_disable_vhost'              => $modsec_disable_vhost,
       'modsec_audit_log_destination'      => $modsec_audit_log_destination,
-      '_modsec_disable_ids'               => $modsec_disable_ids,
+      '_modsec_disable_ids'               => $_modsec_disable_ids,
```

Comment:
The EPP file shouldn't have underscores. What a poor API but I guess 31a137bcb490e6a9041959e8f7115dceac91d4c0 is to blame for that. Not something you need to fix now, but more of a general complaint.